extension/test/data_test.ts

```diff
@@ -69,4 +69,166 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    describe('with a word dict entry', function () {
+      // wordDictEntry was generated with: rcxDict.wordSearch('あ', /* doNames= */ false)
+      const wordDictEntry = {
```

Comment:
This is a definite improvement!

One thing I notice though is there are a lot of fields which seem to not impact the test results but take up a lot of space. These make the test harder to quickly parse for the important inputs.

A reasonable goal is to hide all parts of the input which wouldn't change the outcome of the test. Whatever's left is critical and stands out (and is probably referenced in the test description at least indirectly).

There are at least two options for doing this:

1. If the function/type allows for undefined/null fields then you can just exclude the fields from the object.
2. If there are fields which are required but the value doesn't impact this specific test you can create a default object (or a factory method) that you use as the base for creating your test inputs. (Careful to never actually rely on the default values in assertions!)

    This is pretty easy to do with the spread operator I believe!

In this case, we definitely need `hasNames: false` and the `data` but I'm not sure about stuff like `index` and `matchLen`. It's possible they are derived from `data` so also important. That connection will be hard to show unfortunately so a comment might be the best we can do.